Mathlib/Order/Concept.lean

```diff
@@ -282,6 +307,36 @@ theorem codisjoint_extent_intent [IsTrichotomous α r'] [IsTrans α r'] :
   · contradiction
   · assumption
 
+theorem isCompl_extent_intent [IsStrictTotalOrder α r'] (c' : Concept α α r') :
+    IsCompl c'.extent c'.intent :=
+  ⟨c'.disjoint_extent_intent, c'.codisjoint_extent_intent⟩
+
+/-- See `isLowerSet_extent'` for the theorem on a `<`-concept. -/
+theorem isLowerSet_extent {α : Type*} [Preorder α] (c : Concept α α (· ≤ ·)) :
```

Comment:
Surely this just needs transitivity of <=? Preorder is too strong.

Reply:
That's true, but presumably in any practical instance where you have a transitive ≤ relation you at least have a preorder? In fact, the only occurence of `IsTrans ?_ (· ≤ ·)` throughout Mathlib is the instance that infers this from a `Preorder`.